doc/sphinx/about.rst

```diff
@@ -0,0 +1,97 @@
+.. _sec-about:
+
+*************
+About Cantera
+*************
+
+Cantera is an open source scientific computing project, that allows users to
+quickly and efficiently incorporate and execute fundamental thermodynamic,
+chemical kinetic, and species transport calculations in their work. This can be
+achieved by standalone command-line calls to core Cantera functions,
+incorporating these calls into bespoke code or software, or by using any of the
+solvers or examples distributed with the Cantera software package.
+
+Cantera was originally started by Prof. David G. Goodwin at the California
+Institute of Technology. Building on Prof. Goodwin's legacy, Cantera is licensed
+under a permissive 3-Clause BSD license, ensuring that the software will remain
+available for all to use.
+
+Cantera is developed openly on GitHub. Contributions are welcomed from anyone in
+the community; please see the `Contributors guide
+<https://github.com/Cantera/cantera/blob/master/CONTRIBUTING.md>`_ for
+assistance in getting started.
+
+All online and in-person interactions and communications related to Cantera are
+governed by the `Cantera Code of Conduct
+<https://github.com/Cantera/cantera/blob/master/CODE_OF_CONDUCT.md>`_. This code
+of conduct sets expectations for the community to ensure that users and
+contributors are able to participate in a respectful and welcoming environment.
+
+Steering Committee
+==================
+
+The role of the steering committee is to ensure the long-term health of the
+Cantera project. This includes overseeing the development of the Cantera code
+and community in a way that most benefits the entire community of users and
+contributors. The steering committee can be contacted at `steering@cantera.org
+<mailto:steering@cantera.org>`_ The current steering committee (in alphabetical
+order) is:
+
+    * Steven DeCaluwe, Colorado School of Mines
+    * \C. Franklin Goldsmith, Brown University
```

Comment:
What does the backslash do here?

Reply:
Prevents the `C.` from being interpreted as an ordered sublist